docs/content/server/new_blocks.md

```diff
@@ -129,10 +130,10 @@ from backend.data.model import CredentialsField
 # API Key auth:
 class BlockWithAPIKeyAuth(Block):
     class Input(BlockSchema):
-        credentials = CredentialsField(
+        # Note that the type hint below is require or you will get a type error. 
+        credentials: CredentialsMetaInput[Literal['github'], Literal['api_key']] = CredentialsField(

```

Comment:
It isn't explicitly explained what is happening here, what is the first and second `Literal`?
We could list currently supported providers and credential types.

Reply:
added note